spaceship/lib/spaceship/tunes/app_version.rb

```diff
@@ -702,7 +702,8 @@ def setup_messages_screenshots_for(row)
             screenshot_data = screenshot["value"]
             data = {
                 device_type: display_family['name'],
-                language: row["language"]
+                language: row["language"],
+                platform: "imessage" # to identify imessage screenshots later on (e.g: during download)
```

Comment:
As far as I can tell from this and the other PR, the platform is only ever set to `"imessage"`. In that case I'd prefer to have a boolean flag, e.g. `is_imessage?`